gtsfm/utils/io.py

```diff
@@ -101,6 +101,19 @@ def save_json_file(
         json.dump(data, f, indent=4)
 
 
+def read_json_file(fpath: Union[str, Path]) -> Any:
```

Comment:
We need this wrapper for dask?

Reply:
I included it just to avoid writing `with(blah): json.load()` over and over again.

Purely for convenience.